newIDE/app/src/InstancesEditor/CompactInstancePropertiesEditor/index.js

```diff
@@ -202,6 +212,23 @@ const CompactInstancePropertiesEditor = ({
               onInstancesModified={onInstancesModified}
             />
           </Column>
+          {shouldDisplayTileSetVisualizer && (
+            <>
+              <Column>
+                <Spacer />
```

Comment:
if you put the spacer in the column, you probably don't need the `<>` at the top